setup.cfg

```diff
@@ -94,19 +94,15 @@ runtime =
     json5==0.9.11
     jsonpath-ng==1.5.3
     jsonpath-rw>=1.4.0,<2.0.0
-    # to be removed when https://github.com/python-openapi/openapi-schema-validator/issues/131 is resolved
-    jsonschema<=4.19.0
     moto-ext[all]==5.0.0.post1
     opensearch-py>=2.4.1
     pymongo>=4.2.0
     pyopenssl>=23.0.0
 
-# @deprecated - use extra 'runtime' instead.
-full =
-    %(runtime)s
-
```

Comment:
Are you sure this extra is not used anywhere anymore? Because this would basically break all these usages.

Reply:
I searched for any usage of `make full` in both community and ext with ripgrep and did not find any usage of it there. Is there any other place I could search?

Reply:
I prefer to use the GitHub search for tasks like this: https://github.com/search?q=org%3Alocalstack+%22%5Bfull%5D%22&type=code

Looking good!